.github/workflows/build.yml

```
@@ -173,13 +185,25 @@ jobs:
           Get-FileHash MNE-Python-$env:MNE_INSTALLER_VERSION-Windows.exe -Algorithm SHA256 > MNE-Python-$env:MNE_INSTALLER_VERSION-Windows.exe.sha256.txt
           Get-Content MNE-Python-$env:MNE_INSTALLER_VERSION-Windows.exe.sha256.txt
 
-      - name: Run installer (macOS)
+      - name: Check installer (macOS)
         if: ${{ runner.os == 'macOS' }}
+        env:
+          ARCH: ${{ matrix.arch }}
         run: |
           echo `pwd`
-          installer -verbose -pkginfo -pkg ./MNE-Python-${MNE_INSTALLER_VERSION}-macOS_Intel.pkg
-          installer -verbose -dominfo -pkg ./MNE-Python-${MNE_INSTALLER_VERSION}-macOS_Intel.pkg
-          installer -verbose -volinfo -pkg ./MNE-Python-${MNE_INSTALLER_VERSION}-macOS_Intel.pkg
+          if [[ ${ARCH} == 'native' ]]; then
+            MACOS_SUFFIX="Intel"
+          else
+            MACOS_SUFFIX="ARM"
```

Since the filenames of the installer packages are "user-facing", I'd suggest we use the suffix "M1" for now, as I assume most Apple Silicon users won't really know what "ARM" is supposed to mean.
```suggestion
            MACOS_SUFFIX="M1"
```